tic-tac-toe-app/Cell.jsx

```diff
@@ -0,0 +1,16 @@
+const Cell = (props) => {
+  const [square, setSquare] = React.useState('')
+  
+  //the cell only know its own value
+  const changeCellValue = () => {
+    setSquare(props.turn);
+    props.changeTurn();
+    props.updateBoard(props.id, props.turn);
+  };
```

Comment:
Players may discover that they can "cheat" when playing this app. 😅 Consider how to update this function (or create a new function, if prefered) so that players are restricted from changing their opponent's values.

Reply:
I actually forgot to fix this bug, thank you will update